docs/src/parameter_distributions.md

```diff
@@ -126,15 +128,16 @@ This is simply a `String` used to identify different parameters in multi-paramet
 ## ParameterDistribution constructor
 
 The [Recommended constructor](@ref constrained-gaussian), `constrained_gaussian()`, is described above. For more general cases in which the prior needs to be specified in more detail, a `ParameterDistribution` may be constructed "manually" from its component objects:
-
 ```julia
+using EnsembleKalmanProcesses.ParameterDistributions # for `ParameterDistribution`, `combine_distributions`
 prior_1 = ParameterDistribution(distribution_1, constraint_1, name_1)
 prior_2 = ParameterDistribution(distribution_2, constraint_2, name_2)
 prior = combine_distributions( [prior_1, prior_2])
 ```
```

Comment:
```suggestion
```@example
using EnsembleKalmanProcesses.ParameterDistributions # for `ParameterDistribution`, `combine_distributions`
prior_1 = ParameterDistribution(distribution_1, constraint_1, name_1)
prior_2 = ParameterDistribution(distribution_2, constraint_2, name_2)
prior = combine_distributions( [prior_1, prior_2])
nothing # hide
```
```

I think it would be better to change all of these to `@example`-type blocks (with `nothing # hide` at the end to hide output, where relevant) so that the code is actually run, and will presumably fail if we change the API in the future.

Reply:
looks like the formatting got a bit messed up here, but I hope you get the point. See also the relevant docs [here](https://documenter.juliadocs.org/stable/man/syntax/index.html#@example-block).

Reply:
Oh you want these to be runnable? 

I guess i have no issue with this. In general we tried to avoid it as it can lead to significantly increased docs build times. But these should be small, perhaps once we have good "show" statements we can remove the `nothing` too

Reply:
well.. We don't have to, but that would "automatically" ensure that the examples always work as expected, and as you mention, would nicely tie into the discussion in #257.

Reply:
done.